### PR TITLE
T/174 Fix translations upload to transifex

### LINKED ISSUE #174: Can't upload translations to transifex
```
(master cbe80fd) p@m /workspace/ckeditor5> gulp translations:upload
[10:28:31] Using gulpfile /workspace/ckeditor5/gulpfile.js
[10:28:31] Starting 'translations:upload'...
? Provide the Transifex token (generate it here: https://www.transifex.com/user/settings/api/): ******************************************
[10:28:33] Error: Status code: 400
[10:28:33] 'translations:upload' errored after 1.74 s
[10:28:33] Error: Status code: 400
    at Request.handleJsonResponse [as _callback] (/workspace/ckeditor5-dev/packages/ckeditor5-dev-env/lib/translations/transifex-service.js:109:19)
    at Request.self.callback (/workspace/ckeditor5-dev/node_modules/request/request.js:188:22)
    at emitTwo (events.js:106:13)
    at Request.emit (events.js:191:7)
    at Request.<anonymous> (/workspace/ckeditor5-dev/node_modules/request/request.js:1171:10)
    at emitOne (events.js:96:13)
    at Request.emit (events.js:188:7)
    at IncomingMessage.<anonymous> (/workspace/ckeditor5-dev/node_modules/request/request.js:1091:12)
    at IncomingMessage.g (events.js:291:16)
    at emitNone (events.js:91:20)
```

Something... broke. Not sure what, though.

### PR DESCRIPTION
**Pull request by ma2ciek**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Closes #174.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+// Place your settings in this file to overwrite default and user settings.
```
Comment (Reinmar) [inline .vscode/settings.json]: Ekhm :P

File: packages/ckeditor5-dev-env/tests/translations/upload.js [hunk 2]
```diff
@@ -40,7 +40,7 @@ describe( 'upload', () => {
 
 			fs: {
 				readdirSync: sandbox.spy( () => packageNames ),
-				readFileSync: sandbox.spy( fileName => fileContents[ fileName ] )
+				createReadStream: sandbox.spy( fileName => fileContents[ fileName ] )
```
Comment (ma2ciek) [inline packages/ckeditor5-dev-env/tests/translations/upload.js]: This needs to be improved as this is no longer valid mock.

File: .vscode/settings.json (hunk 1 above)
Comment (ma2ciek) [inline .vscode/settings.json]: no... :D
May I add this file to the .gitignore?

Comment (ma2ciek) [inline .vscode/settings.json]: or just whole .vscode/ directory?

Comment (Reinmar) [inline .vscode/settings.json]: I don't like adding editor specific files to .gitignore (cause we'd need to do that for every editor which creates them) ;| Can't VS be configured to keep those files somewhere else?

Comment (pomek) [inline .vscode/settings.json]: I suggest creating a global `.gitignore`:

1. Create a file `.gituserignore`.
2. Put anything what do you want. E.g. my file:

```
 *.swo
 *.swp
 .DS_Store

 npm-debug.log

 puphpet/config-custom.yaml

 .grunt/
 .idea/
 .vagrant/

 /vendor/
 /node_modules/
 /bower_components/
```
3. Add to your `~/.gitconfig`

```
[core]
    excludesfile = ~/.gituserignore
```

Comment (ma2ciek) [inline .vscode/settings.json]: It sounds great. 👍

Comment (ma2ciek) [inline .vscode/settings.json]: And it works well! Thanks @pomek

File: packages/ckeditor5-dev-env/tests/translations/upload.js (hunk 2 above)
Comment (ma2ciek) [inline packages/ckeditor5-dev-env/tests/translations/upload.js]: Moved this to the https://github.com/ckeditor/ckeditor5-dev/issues/177 issue.

**Pull request merged**